src/components/action-button.js

```diff
@@ -12,14 +12,13 @@ export const runActions = async actionList => {
 export default function ActionButton({ layout, button, Theme, app, context, senseNavigation, element }) {
   const { style, qStateName } = layout;
   const disabled = layout.useEnabledCondition && layout.enabledCondition === 0;
-  const formattedStyles = styleFormatter.getStyles({ style, disabled, Theme, element });
+  const formattedStyles = styleFormatter.getStyles({ style, disabled, Theme, element, button });
```

Comment:
button not used?

Reply:
No, noticed that button was not used in the getStyles function. Maybe the setting of the label was in there before but I don't know

Reply:
So remove the button argument?